admin-dev/themes/default/template/helpers/list/list_action_delete.tpl

```diff
@@ -25,6 +25,16 @@
 <a href="#"
    title="{$action|escape:'html':'UTF-8'}"
    class="delete"
-   onclick="{if $confirm}confirm_link('', '{$confirm|escape:'html':'UTF-8'}', '{l s='Yes' d='Admin.Global'}', '{l s='No' d='Admin.Global'}', '{$href|escape:'html':'UTF-8'}', '#'){else}event.stopPropagation();event.preventDefault(){/if}">
+   onclick="
+    {if $confirm && !isset($is_images_page)}
```

Comment:
Every legacy controller may override this file, if you need something specific to this controller, you need to override it instead of making conditions like this 👍

Reply:
One example is Attribute Groups controller:
`admin-dev/themes/default/template/controllers/attributes_groups/helpers/list/`

there's a lot more